README.rst

```diff
@@ -73,6 +73,65 @@ Run `vagrant ssh web`, that should put you in the `/vagrant` directory.
 While there run `openssl req -newkey rsa:2048 -sha256 -nodes -keyout bmo-web.vm-key.pem -x509 -days 365 -out bmo-web.vm.pem`.
 Restart morbo and you should now be able to visit https://bmo-web.vm.
 
+Vagrant and Windows Hyper-Vagrant
+---------------------------------
+
+Enable hyperv support in Windows 10
+
+- Right click on the Windows button and select ‘Apps and Features’.
+- Select Turn Windows Features on or off.
+- Select Hyper-V and click OK.
+- May need to reboot.
+
+Run PowerShell as Administrator
+
+- Press the Windows key.
+- Type `powershell` in the search field.
+- In the search results, select `Run as administrator`.
+
+Create NAT Switch with IP address of 192.168.3.1/24
+
+.. code-block:: powershell
+
+  New-VMSwitch -SwitchName “VagrantSwitch” -SwitchType Internal
+  New-NetIPAddress -IPAddress 192.168.3.1 -PrefixLength 24 -InterfaceAlias “vEthernet (VagrantSwitch)”
+  New-NetNAT -Name “NATNetwork” -InternalIPInterfaceAddressPrefix 192.168.3.0/24
+
+Install DHCP Servor for Windows
+
+- http://www.dhcpserver.de/cms/
+- Configure to serve IP addresses to the NAT Switch network above.
+- Set the IP range to be 100-254
+
+Install Vagrant for Windows
+
+Install Git for Windows
+
+Set some environment variables to use for Vagrant
+
+.. code-block:: powershell
+
+  [System.Environment]::SetEnvironmentVariable('HYPERV', '1')
+
+Update hosts file in Windows
+
+- Press the Windows key.
+- Type `notepad` in the search field.
+- In the search results, select `Run as administrator`.
+- From Notepad, open the following file: ``C:\Windows\System32\Drivers\etc\hosts``.
+- Add ``192.168.3.43 bmo-web.vm`` to the end of the file.
+- Select File > Save to save your changes.
+
+Git clone BMO repo ``git clone https://github.com/mozilla-bteam/bmo bmo``
+
+.. code-block:: bash
+
+  cd bmo
+  vagrant up --provider hyperv
```

Comment:
Got as far as getting this going and then failed:
```
==> db: Running provisioner: main (ansible_local)...
    db: Installing Ansible...
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

sudo rpm -i https://dl.fedoraproject.org/pub/epel/epel-release-latest-`rpm -E %dist | sed -n 's/.*el\([0-9]\).*/\1/p'`.noarch.rpm

Stdout from the command:

Stderr from the command:

curl: (6) Couldn't resolve host 'dl.fedoraproject.org'
error: skipping https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm - transfer failed

mozilla@DESKTOP-20HKSV2 MINGW64 ~/work/newbz/bmo (testhyperv)
$ vagrant ssh db
[vagrant@bmo-db ~]$ ping www.google.com
ping: unknown host www.google.com
[vagrant@bmo-db ~]$ ip a
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
    inet6 ::1/128 scope host
       valid_lft forever preferred_lft forever
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP qlen 1000
    link/ether 00:15:5d:89:01:04 brd ff:ff:ff:ff:ff:ff
    inet6 fe80::215:5dff:fe89:104/64 scope link
       valid_lft forever preferred_lft forever
[vagrant@bmo-db ~]$
```


![snipdebug](https://user-images.githubusercontent.com/7828780/70854890-ff43a000-1e8f-11ea-8a0c-c0692805ecec.PNG)

Also tried different settings on the DHCP server thing like setting the gateway to my router gateway ip. Doing so i get this from `ip a` (the correct 192.168.3.100 address) but still can't connect to the internet from the vm

```
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP qlen 1000
    link/ether 00:15:5d:89:01:06 brd ff:ff:ff:ff:ff:ff
    inet 192.168.3.100/24 brd 192.168.3.255 scope global eth0
    inet6 fe80::215:5dff:fe89:106/64 scope link
       valid_lft forever preferred_lft forever
```
Also tried installing it as a service and it still doesn't work (dunno if it is this part that causes the internet connection, assuming so)